lib/spack/docs/build_systems/multiplepackage.rst

```diff
@@ -1,350 +0,0 @@
-.. Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
I'm guessing you moved all documentation on packages with multiple build systems to the Packaging Guide? I would prefer keeping this in the build system docs. The Packaging Guide is a bit monolithic and takes forever to find things. Even if the docs end up being duplicated, I think people will first look to the Build System docs to understand how to support 2 build systems. Although maybe I'm biased because I wrote these docs.

Reply:
Honestly I deleted this since the docs (which are great at the moment), are enumerating all the workarounds we have to deal with multiple build-systems. With this PR there will be only one correct way to deal with them, which is inherit from multiple `*Package` base classes and implement the customization a software needs to the corresponding builders.